src/test/java/base/BaseTest.java

```diff
@@ -2,11 +2,36 @@
 
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.testng.annotations.*;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+
 import java.time.Duration;
 
 public class BaseTest {
-
     protected WebDriver driver;
+    protected WebDriverWait wait;
+
+    @BeforeClass
+    public void setupClass() {
+        WebDriverManager.chromedriver().setup();
```

Comment:
Why is this before class and the rest of driver configuration inside beforeMethod?

Why do we init driver before each method but close it only after class? so if there are 10 tests in a class, we will set up driver 10 time but close it once? 

How often should we set up and close driver for a test?